### PR TITLE
Added loopbacks on octaves runcard

### PR DESCRIPTION
**Pull request by jordivallsq**
In order to demodulate with other octave ports rather than port 1 the loopbacks have to be added based on https://docs.quantum-machines.co/1.1.7/qm-qua-sdk/docs/Guides/octave/#the-loopbacks-key and following the octave block diagram 
octave_user_diagram-1

### CODE REVIEW

File: docs/releases/changelog-dev.md [hunk 1]
```diff
@@ -57,6 +57,25 @@
 
   [#762](https://github.com/qilimanjaro-tech/qililab/pull/762)
 
+- Added loopbacks in the octave config file for qua following the documentation at https://docs.quantum-machines.co/1.2.0/qm-qua-sdk/docs/Guides/octave/?h=octaves#setting-the-octaves-clock. By default only port 1 of the octave is linked with a local demodulator, to work with the rest of the ports at the back ports must be connected based on the Octave Block Diagram [https://docs.quantum-machines.co/1.2.0/qm-qua-sdk/docs/Hardware/octave/#octave-block-diagram]. Where `Synth` is one of the possible 3 synths and `Dmd` is one of the 2 demodulators.
```
Comment (GuillermoAbadLopez) [inline docs/releases/changelog-dev.md:60]: Nice chengelog!

Screenshot 2024-08-07 at 12 42 47

**Review by GuillermoAbadLopez — APPROVED** [review]
Looks very nice! GJ

But please do test it on HW before merging! 🙌

File: docs/releases/changelog-dev.md (hunk 1 above)
Comment (jordivallsq) [reply docs/releases/changelog-dev.md:60]: Nice!!

**Comment by jordivallsq** [comment]
I will test with HW and merge

**Comment by jordivallsq** [comment]
The Loopback has been tested in QUA following QM documentation

**Pull request merged**